contracts/utils/UniversalResolver.sol

```diff
@@ -46,6 +62,11 @@ struct OffchainLookupExtraData {
     bytes data;
 }
 
+struct Result {
```

Comment:
Any reason not to roll the resolver address into this struct - or alternately, to skip the struct and return the bool as part of the return tuple?

Reply:
the resolver address will be the same for every result returned from a multicall, but the success can potentially be different. for single calls the `Result` is unwrapped before returning `(bytes, address)`, reverting if unsuccessful.

Reply:
![Screenshot_20231122_024004_Binance.jpg](https://github.com/ensdomains/ens-contracts/assets/117018255/7865c0ac-b769-4100-b0e3-a6a3e5bf7467)

![Screenshot_20231114_014741_Chrome.jpg](https://github.com/ensdomains/ens-contracts/assets/117018255/bfbedf9c-63c6-46ef-b03f-741f48ccb4e5)